orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/expressions/functions/UrlExpressionFunctionProvider.java

```diff
@@ -81,6 +78,10 @@ public Functions getFunctions() {
         new FunctionDefinition(
             "readYaml",
             "Parses YAML from a string to be accessed, parsed JSON can be accessed as an object",
+            new FunctionParameter(String.class, "value", "A String containing YAML text")),
+        new FunctionDefinition(
+            "readYamlAll",
+            "Parses multi-doc YAML from a string to be accessed, list of parsed JSON can be accessed as objects",
```

Comment:
```suggestion
            "Parses multi-doc YAML from a string to be accessed, list of parsed YAML can be accessed as objects",
```
(it looks like the text for readYaml was copy+pasted from readJson, so maybe it can be fixed too)

Reply:
Fixed